### ISSUE TITLE
Notices not shown in msg

### ISSUE THREAD

**Issue by petski** [ercpe/smart-check#1]
If there are only notices for a disk, the msg will always display: "device-model: S.M.A.R.T. data OK", even though --exclude-notices is left to the default (False).

I propose this patch in `src/smartcheck/main.py`, on line `83`

```
- if any((ae.level == AttributeWarning.Warning for ae in attribute_errors.values())):
+ if any((ae.level in [ AttributeWarning.Warning, AttributeWarning.Notice ] for ae in attribute_errors.values())):
```

**Comment by ercpe** [comment]
Thanks for the report, I will look at it on the weekend.
Could you please attach a full `smartctl -a` report?

**Comment by petski** [comment]
Thanks in advance. Find output attached.
[smartctl_a.txt](https://github.com/ercpe/smart-check/files/558355/smartctl_a.txt)

**Comment by ercpe** [comment]
Fixed in 0.3.1. Thank you for reporting this issue.
